### PR TITLE
MC-18508: add API docs for creating a resource commitment

### PR DESCRIPTION
**Pull request by dpmurray21**
### Fixes [MC-18508](https://cloud-ops.atlassian.net/browse/MC-18508)

API docs for creating a commitment

https://github.com/cloudops/cloudmc-core/pull/2222

### CODE REVIEW

**Review by m-orfanos — APPROVED** [review]

File: source/includes/administration/_commitments.md [hunk 1]
```diff
@@ -227,3 +226,168 @@ Attributes | &nbsp;
...
+   }
+}
+
+```
+
+
+Attributes | &nbsp;
+---- | ---
+`name`<br/>*string* | The name of the commitment.
+`organization`<br/>*Organization* | The organization that owns the commitment.
```
Comment (jasondias9) [inline source/includes/administration/_commitments.md]: is this the org that owns the commitment or the org that this commitment is assigned to? should there be both?

Comment (dpmurray21) [reply source/includes/administration/_commitments.md]: 5ad969612016f04f269caf8b52dc10a4fe0a2d2e

**Review by jasondias9 — COMMENTED** [review]
Should we add the reseller field in this PR as well? https://github.com/cloudops/cloudmc-core/commit/b5d74231ab4b7dd3ae58fb594f94a2f2a8301b1b

**Comment by dpmurray21** [comment]
> Should we add the reseller field in this PR as well? [cloudops/cloudmc-core@b5d7423](https://github.com/cloudops/cloudmc-core/commit/b5d74231ab4b7dd3ae58fb594f94a2f2a8301b1b)

3be49065eb903bd8816bc9e25f2a3dfbe7c870d6

**Review by jasondias9 — APPROVED** [review]

**Pull request merged**